onnx/defs/nn/defs.cc

```diff
@@ -40,8 +40,8 @@ void convPoolShapeInference(
   }
 
   auto input_shape = ctx.getInputType(input1Idx)->tensor_type().shape();
-  if (input_shape.dim_size() < 2) {
-    fail_shape_inference("Input tensor must have atleast 2 dimensions");
+  if (input_shape.dim_size() < 3) {
+    fail_shape_inference("Input tensor must have at least 3 dimensions");
```

Comment:
[nit] It'd be nice if you can do something like 
```suggestion
    fail_shape_inference("Input tensor must have at least %d dimensions", number_of_max_conv_rnn_input_dims);
```
My syntax is definitely wrong. Just want to have a concrete example for things in my mind.